blocks/amp-story/amp-story-grid-layer.js

```diff
@@ -0,0 +1,124 @@
+const { __ } = wp.i18n;
+const {
+	registerBlockType
+} = wp.blocks;
+const {
+	InspectorControls,
+	InnerBlocks
+} = wp.editor;
+const {
+	SelectControl
+} = wp.components;
+
+const ALLOWED_BLOCKS = [
+	'core/code',
```

Comment:
Removed `button` from the list since according to the [specs](https://www.ampproject.org/docs/reference/components/amp-story#children) it seems that the grid layer does not allow buttons/links, only the CTA layer does.